src/viewport/old-viewport.js

```diff
@@ -337,8 +337,8 @@ export default class Viewport {
       this._createMat4(),
       2 * Math.atan((this.height / 2) / this.altitude), // fov in radians
       this.width / this.height,                         // aspect ratio
-      0.1,                                              // near plane
-      this.farZ * 10.0                                  // far plane
+      0.2,                                              // near plane
+      this.farZ * 2                                     // far plane
```

Comment:
> Any specific reason that we change this?

Turned out that the Voronoi layer is extremely sensitive to rounding errors in the depth buffer, and we get some "flickering" while tilting (not panning). Reducing the [range of the depth buffer](https://www.opengl.org/wiki/Depth_Buffer_Precision) from 100 to 10 mitigates this noticeably. On the other hand, the flickering is still not completely solved, and we can hardly reduce the depth buffer range more than that, so it doesn't really matter.

Also please note that this change is in unused code. We are importing the `Viewport` from viewport-mercator-project, not from this file. I was just experimenting here, so others can play with it after we land.